examples/transform/quip_example.py

```diff
@@ -0,0 +1,108 @@
+"""
+WARNING: This example requires the following minimum versions:
+    * compressed-tensors>=0.10.3.dev
+    * transformers>=4.56.dev
+Note that (you may need to install from source)
+
+Models produced by this example will not be runnable in vLLM without
+the following changes: https://github.com/vllm-project/vllm/pull/22219
+"""
+
+from datasets import load_dataset
+from packaging import version
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from transformers.utils.import_utils import _is_package_available
+
+from llmcompressor import oneshot
+from llmcompressor.modifiers.quantization import QuantizationModifier
+from llmcompressor.modifiers.transform import QuIPModifier
+from llmcompressor.utils import dispatch_for_generation
+
+# check correct versioning
+_, ct_version = _is_package_available("compressed_tensors", return_version=True)
```

Comment:
We should just remove these and assume ct has been updated. This is not needed on main

Reply:
We should remove these once CT releases